test/jdk/tools/jpackage/windows/WinChildProcessTest.java

```diff
@@ -68,21 +68,21 @@ public static void test() throws Throwable {
                     .execute(0).getOutput();
             String pidStr = output.get(0);
 
-            // parse calculator PID
-            calcPid = Long.parseLong(pidStr.split("=", 2)[1]);
+            // parse child PID
+            childPid = Long.parseLong(pidStr.split("=", 2)[1]);
 
             // Check whether the termination of third party application launcher
             // also terminating the launched third party application
             // If third party application is not terminated the test is
             // successful else failure
-            Optional<ProcessHandle> processHandle = ProcessHandle.of(calcPid);
+            Optional<ProcessHandle> processHandle = ProcessHandle.of(childPid);
             boolean isAlive = processHandle.isPresent()
                     && processHandle.get().isAlive();
             System.out.println("Is Alive " + isAlive);
```

Comment:
This log statement is redundant. The following `TKit.assertTrue()` call produces enough information in the log:
```
[03:45:39.326] TRACE: assertEquals(0): Check command [test\output\WinChildProcessTest\WinChildProcessTest.exe](1) exited with 0 code
Is Alive true
[03:45:39.326] TRACE: assertTrue(): Check is child process is alive
```
It is irrelevant to this PR and is an oversight from the previous review. However, I believe it is OK to make this small change in this PR.